From 90c7a2be8c6f4cd0351433c771bcf1c146d646d4 Mon Sep 17 00:00:00 2001
From: danielk43 <11856117+danielk43@users.noreply.github.com>
Date: Mon, 13 Dec 2021 09:20:30 -0500
Subject: [PATCH 1/2] Remove fp

---
 target/product/generic_system.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/target/product/generic_system.mk b/target/product/generic_system.mk
--- a/target/product/generic_system.mk
+++ b/target/product/generic_system.mk
@@ -63,7 +63,6 @@ PRODUCT_PACKAGES += libfwdlockengine
 # System libraries commonly depended on by things on the system_ext or product partitions.
 # These lists will be pruned periodically.
 PRODUCT_PACKAGES += \
-    android.hardware.biometrics.fingerprint@2.1 \
     android.hardware.radio@1.0 \
     android.hardware.radio@1.1 \
     android.hardware.radio@1.2 \

