From 691d871bf6ce24d658408422b4f652448d774525 Mon Sep 17 00:00:00 2001
From: danielk43 <11856117+danielk43@users.noreply.github.com>
Date: Sun, 17 Oct 2021 12:07:45 -0400
Subject: [PATCH] Alias signify cmd if applicable

---
 signify_prehash.sh | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/signify_prehash.sh b/signify_prehash.sh
--- a/signify_prehash.sh
+++ b/signify_prehash.sh
@@ -9,5 +9,8 @@ source "$(dirname ${BASH_SOURCE[0]})/common.sh"
 key="$(realpath $1)"
 file=$(basename $2)
 
+shopt -s expand_aliases
+[[ -n $(command -v signify-openbsd) ]] && alias signify="signify-openbsd"
+
 cd "$(dirname $2)"
 sha256sum --tag "$file" | signify -S -s "$key" -e -m - -x "$file.sig"

