From ca6acbce6e56758907ac76e0119d169bc7e1a3fe Mon Sep 17 00:00:00 2001
From: danielk43 <11856117+danielk43@users.noreply.github.com>
Date: Fri, 22 Dec 2023 17:11:38 -0500
Subject: [PATCH 1/2] raven: Update priv-apps

---
 Android.bp      | 333 ------------------------------------------------
 raven-vendor.mk |  45 -------
 2 files changed, 378 deletions(-)

diff --git a/Android.bp b/Android.bp
index c594a39..de05f78 100644
--- a/Android.bp
+++ b/Android.bp
@@ -27,30 +27,6 @@ android_app_import {
 	product_specific: true,
 }
 
-android_app_import {
-	name: "AppDirectedSMSService",
-	owner: "google",
-	apk: "proprietary/product/priv-app/AppDirectedSMSService/AppDirectedSMSService.apk",
-	presigned: true,
-	dex_preopt: {
-		enabled: false,
-	},
-	privileged: true,
-	product_specific: true,
-}
-
-android_app_import {
-	name: "CarrierServices",
-	owner: "google",
-	apk: "proprietary/product/priv-app/CarrierServices/CarrierServices.apk",
-	presigned: true,
-	dex_preopt: {
-		enabled: false,
-	},
-	privileged: true,
-	product_specific: true,
-}
-
 android_app_import {
 	name: "CarrierSettings",
 	owner: "google",
@@ -63,174 +39,6 @@ android_app_import {
 	product_specific: true,
 }
 
-android_app_import {
-	name: "CarrierWifi",
-	owner: "google",
-	apk: "proprietary/product/priv-app/CarrierWifi/CarrierWifi.apk",
-	presigned: true,
-	dex_preopt: {
-		enabled: false,
-	},
-	privileged: true,
-	product_specific: true,
-}
-
-android_app_import {
-	name: "ConnMO",
-	owner: "google",
-	apk: "proprietary/product/priv-app/ConnMO/ConnMO.apk",
-	certificate: "platform",
-	dex_preopt: {
-		enabled: false,
-	},
-	privileged: true,
-	product_specific: true,
-}
-
-android_app_import {
-	name: "DCMO",
-	owner: "google",
-	apk: "proprietary/product/priv-app/DCMO/DCMO.apk",
-	certificate: "platform",
-	dex_preopt: {
-		enabled: false,
-	},
-	privileged: true,
-	product_specific: true,
-}
-
-android_app_import {
-	name: "DMService",
-	owner: "google",
-	apk: "proprietary/product/priv-app/DMService/DMService.apk",
-	certificate: "platform",
-	dex_preopt: {
-		enabled: false,
-	},
-	privileged: true,
-	product_specific: true,
-}
-
-android_app_import {
-	name: "DeviceIntelligenceNetworkPrebuilt",
-	owner: "google",
-	apk: "proprietary/product/priv-app/DeviceIntelligenceNetworkPrebuilt/DeviceIntelligenceNetworkPrebuilt.apk",
-	presigned: true,
-	dex_preopt: {
-		enabled: false,
-	},
-	privileged: true,
-	product_specific: true,
-}
-
-android_app_import {
-	name: "DevicePersonalizationPrebuiltPixel2021",
-	owner: "google",
-	apk: "proprietary/product/priv-app/DevicePersonalizationPrebuiltPixel2021/DevicePersonalizationPrebuiltPixel2021.apk",
-	presigned: true,
-	dex_preopt: {
-		enabled: false,
-	},
-	privileged: true,
-	product_specific: true,
-}
-
-android_app_import {
-	name: "DiagMon",
-	owner: "google",
-	apk: "proprietary/product/priv-app/DiagMon/DiagMon.apk",
-	certificate: "platform",
-	dex_preopt: {
-		enabled: false,
-	},
-	privileged: true,
-	product_specific: true,
-}
-
-android_app_import {
-	name: "EuiccGoogle",
-	owner: "google",
-	apk: "proprietary/product/priv-app/EuiccGoogle/EuiccGoogle.apk",
-	presigned: true,
-	dex_preopt: {
-		enabled: false,
-	},
-	privileged: true,
-	product_specific: true,
-}
-
-android_app_import {
-	name: "HardwareInfo",
-	owner: "google",
-	apk: "proprietary/product/priv-app/HardwareInfo/HardwareInfo.apk",
-	presigned: true,
-	dex_preopt: {
-		enabled: false,
-	},
-	privileged: true,
-	product_specific: true,
-}
-
-android_app_import {
-	name: "HotwordEnrollmentOKGoogleFUSION",
-	owner: "google",
-	apk: "proprietary/product/priv-app/HotwordEnrollmentOKGoogleFUSION/HotwordEnrollmentOKGoogleFUSION.apk",
-	presigned: true,
-	dex_preopt: {
-		enabled: false,
-	},
-	privileged: true,
-	product_specific: true,
-}
-
-android_app_import {
-	name: "HotwordEnrollmentXGoogleFUSION",
-	owner: "google",
-	apk: "proprietary/product/priv-app/HotwordEnrollmentXGoogleFUSION/HotwordEnrollmentXGoogleFUSION.apk",
-	presigned: true,
-	dex_preopt: {
-		enabled: false,
-	},
-	privileged: true,
-	product_specific: true,
-}
-
-android_app_import {
-	name: "OemDmTrigger",
-	owner: "google",
-	apk: "proprietary/product/priv-app/OemDmTrigger/OemDmTrigger.apk",
-	certificate: "platform",
-	dex_preopt: {
-		enabled: false,
-	},
-	privileged: true,
-	product_specific: true,
-}
-
-android_app_import {
-	name: "TetheringEntitlement",
-	owner: "google",
-	apk: "proprietary/product/priv-app/TetheringEntitlement/TetheringEntitlement.apk",
-	certificate: "platform",
-	dex_preopt: {
-		enabled: false,
-	},
-	privileged: true,
-	product_specific: true,
-}
-
-android_app_import {
-	name: "WfcActivation",
-	owner: "google",
-	apk: "proprietary/product/priv-app/WfcActivation/WfcActivation.apk",
-	presigned: true,
-	dex_preopt: {
-		enabled: false,
-	},
-	privileged: true,
-	product_specific: true,
-}
-
 android_app_import {
 	name: "OemRilHookService",
 	owner: "google",
@@ -242,102 +50,6 @@ android_app_import {
 	system_ext_specific: true,
 }
 
-android_app_import {
-	name: "CarrierSetup",
-	owner: "google",
-	apk: "proprietary/system_ext/priv-app/CarrierSetup/CarrierSetup.apk",
-	presigned: true,
-	dex_preopt: {
-		enabled: false,
-	},
-	privileged: true,
-	system_ext_specific: true,
-}
-
-android_app_import {
-	name: "ConnectivityThermalPowerManager",
-	owner: "google",
-	apk: "proprietary/system_ext/priv-app/ConnectivityThermalPowerManager/ConnectivityThermalPowerManager.apk",
-	certificate: "platform",
-	dex_preopt: {
-		enabled: false,
-	},
-	privileged: true,
-	system_ext_specific: true,
-}
-
-android_app_import {
-	name: "EuiccGoogleOverlay",
-	owner: "google",
-	apk: "proprietary/system_ext/priv-app/EuiccGoogleOverlay/EuiccGoogleOverlay.apk",
-	certificate: "platform",
-	dex_preopt: {
-		enabled: false,
-	},
-	privileged: true,
-	system_ext_specific: true,
-}
-
-android_app_import {
-	name: "EuiccSupportPixel",
-	owner: "google",
-	apk: "proprietary/system_ext/priv-app/EuiccSupportPixel/EuiccSupportPixel.apk",
-	presigned: true,
-	dex_preopt: {
-		enabled: false,
-	},
-	privileged: true,
-	system_ext_specific: true,
-}
-
-android_app_import {
-	name: "EuiccSupportPixelPermissions",
-	owner: "google",
-	apk: "proprietary/system_ext/priv-app/EuiccSupportPixelPermissions/EuiccSupportPixelPermissions.apk",
-	presigned: true,
-	dex_preopt: {
-		enabled: false,
-	},
-	privileged: true,
-	system_ext_specific: true,
-}
-
-android_app_import {
-	name: "HbmSVManager",
-	owner: "google",
-	apk: "proprietary/system_ext/priv-app/HbmSVManager/HbmSVManager.apk",
-	certificate: "platform",
-	dex_preopt: {
-		enabled: false,
-	},
-	privileged: true,
-	system_ext_specific: true,
-}
-
-android_app_import {
-	name: "LLKAgent",
-	owner: "google",
-	apk: "proprietary/system_ext/priv-app/LLKAgent/LLKAgent.apk",
-	certificate: "platform",
-	dex_preopt: {
-		enabled: false,
-	},
-	privileged: true,
-	system_ext_specific: true,
-}
-
-android_app_import {
-	name: "MyVerizonServices",
-	owner: "google",
-	apk: "proprietary/system_ext/priv-app/MyVerizonServices/MyVerizonServices.apk",
-	presigned: true,
-	dex_preopt: {
-		enabled: false,
-	},
-	privileged: true,
-	system_ext_specific: true,
-}
-
 android_app_import {
 	name: "OemRilService",
 	owner: "google",
@@ -386,42 +98,6 @@ android_app_import {
 	system_ext_specific: true,
 }
 
-android_app_import {
-	name: "ShannonQualifiedNetworksService",
-	owner: "google",
-	apk: "proprietary/system_ext/priv-app/ShannonQualifiedNetworksService/ShannonQualifiedNetworksService.apk",
-	presigned: true,
-	dex_preopt: {
-		enabled: false,
-	},
-	privileged: true,
-	system_ext_specific: true,
-}
-
-android_app_import {
-	name: "ShannonRcs",
-	owner: "google",
-	apk: "proprietary/system_ext/priv-app/ShannonRcs/ShannonRcs.apk",
-	presigned: true,
-	dex_preopt: {
-		enabled: false,
-	},
-	privileged: true,
-	system_ext_specific: true,
-}
-
-android_app_import {
-	name: "UwbVendorService",
-	owner: "google",
-	apk: "proprietary/system_ext/priv-app/UwbVendorService/UwbVendorService.apk",
-	presigned: true,
-	dex_preopt: {
-		enabled: false,
-	},
-	privileged: true,
-	system_ext_specific: true,
-}
-
 android_app_import {
 	name: "grilservice",
 	owner: "google",
@@ -583,15 +259,6 @@ prebuilt_etc_xml {
 	soc_specific: true,
 }
 
-prebuilt_etc_xml {
-	name: "fingerprint-goodix",
-	owner: "google",
-	src: "proprietary/vendor/etc/vintf/manifest/fingerprint-goodix.xml",
-	filename_from_src: true,
-	sub_dir: "vintf/manifest",
-	soc_specific: true,
-}
-
 prebuilt_etc_xml {
 	name: "manifest_android.hardware.drm-service.widevine",
 	owner: "google",
diff --git a/raven-vendor.mk b/raven-vendor.mk
index b675fe0..4d5f9d0 100644
--- a/raven-vendor.mk
+++ b/raven-vendor.mk
@@ -44,13 +44,8 @@ PRODUCT_COPY_FILES += \
     vendor/google/raven/proprietary/system_ext/etc/permissions/com.android.hotwordenrollment.common.util.xml:$(TARGET_COPY_OUT_SYSTEM_EXT)/etc/permissions/com.android.hotwordenrollment.common.util.xml \
     vendor/google/raven/proprietary/system_ext/etc/permissions/com.android.omadm.radioconfig.xml:$(TARGET_COPY_OUT_SYSTEM_EXT)/etc/permissions/com.android.omadm.radioconfig.xml \
     vendor/google/raven/proprietary/system_ext/etc/permissions/com.google.android.camera.extensions.xml:$(TARGET_COPY_OUT_SYSTEM_EXT)/etc/permissions/com.google.android.camera.extensions.xml \
-    vendor/google/raven/proprietary/system_ext/etc/permissions/com.google.euiccpixel.permissions.xml:$(TARGET_COPY_OUT_SYSTEM_EXT)/etc/permissions/com.google.euiccpixel.permissions.xml \
-    vendor/google/raven/proprietary/system_ext/etc/permissions/com.google.euiccpixel.xml:$(TARGET_COPY_OUT_SYSTEM_EXT)/etc/permissions/com.google.euiccpixel.xml \
     vendor/google/raven/proprietary/system_ext/etc/permissions/com.samsung.slsi.telephony.oemril.xml:$(TARGET_COPY_OUT_SYSTEM_EXT)/etc/permissions/com.samsung.slsi.telephony.oemril.xml \
     vendor/google/raven/proprietary/system_ext/etc/permissions/com.shannon.imsservice.xml:$(TARGET_COPY_OUT_SYSTEM_EXT)/etc/permissions/com.shannon.imsservice.xml \
-    vendor/google/raven/proprietary/system_ext/etc/permissions/com.shannon.qualifiednetworksservice.xml:$(TARGET_COPY_OUT_SYSTEM_EXT)/etc/permissions/com.shannon.qualifiednetworksservice.xml \
-    vendor/google/raven/proprietary/system_ext/etc/permissions/com.shannon.rcsservice.xml:$(TARGET_COPY_OUT_SYSTEM_EXT)/etc/permissions/com.shannon.rcsservice.xml \
-    vendor/google/raven/proprietary/system_ext/etc/permissions/com.verizon.llkagent.xml:$(TARGET_COPY_OUT_SYSTEM_EXT)/etc/permissions/com.verizon.llkagent.xml \
     vendor/google/raven/proprietary/system_ext/etc/permissions/google-ril.xml:$(TARGET_COPY_OUT_SYSTEM_EXT)/etc/permissions/google-ril.xml \
     vendor/google/raven/proprietary/system_ext/etc/permissions/oemrilhook.xml:$(TARGET_COPY_OUT_SYSTEM_EXT)/etc/permissions/oemrilhook.xml \
     vendor/google/raven/proprietary/system_ext/etc/permissions/privapp-permissions-google-se-lineage.xml:$(TARGET_COPY_OUT_SYSTEM_EXT)/etc/permissions/privapp-permissions-google-se-lineage.xml \
@@ -61,57 +56,22 @@ PRODUCT_COPY_FILES += \
     vendor/google/raven/proprietary/system_ext/lib/vendor.samsung_slsi.telephony.hardware.radioExternal@1.0.so:$(TARGET_COPY_OUT_SYSTEM_EXT)/lib/vendor.samsung_slsi.telephony.hardware.radioExternal@1.0.so \
     vendor/google/raven/proprietary/system_ext/lib64/com.google.edgetpu_app_service-V2-ndk.so:$(TARGET_COPY_OUT_SYSTEM_EXT)/lib64/com.google.edgetpu_app_service-V2-ndk.so \
     vendor/google/raven/proprietary/system_ext/lib64/com.google.edgetpu_vendor_service-V2-ndk.so:$(TARGET_COPY_OUT_SYSTEM_EXT)/lib64/com.google.edgetpu_vendor_service-V2-ndk.so \
-    vendor/google/raven/proprietary/system_ext/lib64/libhbmsvmanager_jni.so:$(TARGET_COPY_OUT_SYSTEM_EXT)/lib64/libhbmsvmanager_jni.so \
     vendor/google/raven/proprietary/system_ext/lib64/libmediaadaptor.so:$(TARGET_COPY_OUT_SYSTEM_EXT)/lib64/libmediaadaptor.so \
     vendor/google/raven/proprietary/system_ext/lib64/libpowerstatshaldataprovider.so:$(TARGET_COPY_OUT_SYSTEM_EXT)/lib64/libpowerstatshaldataprovider.so \
     vendor/google/raven/proprietary/system_ext/lib64/vendor.samsung_slsi.telephony.hardware.oemservice@1.0.so:$(TARGET_COPY_OUT_SYSTEM_EXT)/lib64/vendor.samsung_slsi.telephony.hardware.oemservice@1.0.so \
     vendor/google/raven/proprietary/system_ext/lib64/vendor.samsung_slsi.telephony.hardware.radioExternal@1.0.so:$(TARGET_COPY_OUT_SYSTEM_EXT)/lib64/vendor.samsung_slsi.telephony.hardware.radioExternal@1.0.so \
-    vendor/google/raven/proprietary/system_ext/priv-app/EuiccSupportPixel/DKA_RC5_DVT.in:$(TARGET_COPY_OUT_SYSTEM_EXT)/priv-app/EuiccSupportPixel/DKA_RC5_DVT.in \
-    vendor/google/raven/proprietary/system_ext/priv-app/EuiccSupportPixel/DKA_RC7_DVT.up:$(TARGET_COPY_OUT_SYSTEM_EXT)/priv-app/EuiccSupportPixel/DKA_RC7_DVT.up \
-    vendor/google/raven/proprietary/system_ext/priv-app/EuiccSupportPixel/DKA_RC7_EVT.up:$(TARGET_COPY_OUT_SYSTEM_EXT)/priv-app/EuiccSupportPixel/DKA_RC7_EVT.up \
-    vendor/google/raven/proprietary/system_ext/priv-app/EuiccSupportPixel/esim-full-ofl-v1.img:$(TARGET_COPY_OUT_SYSTEM_EXT)/priv-app/EuiccSupportPixel/esim-full-ofl-v1.img \
-    vendor/google/raven/proprietary/system_ext/priv-app/EuiccSupportPixel/esim-full-v1.img:$(TARGET_COPY_OUT_SYSTEM_EXT)/priv-app/EuiccSupportPixel/esim-full-v1.img
 
 PRODUCT_PACKAGES += \
     PixelCameraServicesRaven \
     VZWAPNLib \
-    AppDirectedSMSService \
-    CarrierServices \
     CarrierSettings \
-    CarrierWifi \
-    ConnMO \
-    DCMO \
-    DMService \
-    DeviceIntelligenceNetworkPrebuilt \
-    DevicePersonalizationPrebuiltPixel2021 \
-    DiagMon \
-    EuiccGoogle \
-    HardwareInfo \
-    HotwordEnrollmentOKGoogleFUSION \
-    HotwordEnrollmentXGoogleFUSION \
-    OemDmTrigger \
-    TetheringEntitlement \
-    WfcActivation \
-    OemRilHookService \
-    CarrierSetup \
-    ConnectivityThermalPowerManager \
-    EuiccGoogleOverlay \
-    EuiccSupportPixel \
-    EuiccSupportPixelPermissions \
-    HbmSVManager \
-    LLKAgent \
-    MyVerizonServices \
     OemRilService \
     PixelNfc \
     RilConfigService \
     ShannonIms \
-    ShannonQualifiedNetworksService \
-    ShannonRcs \
-    UwbVendorService \
     grilservice \
     libhwinfo \
     RadioConfigLib \
-    com.android.hotwordenrollment.common.util \
     com.google.android.camera.extensions \
     google-ril \
     oemrilhook \
@@ -358,7 +318,6 @@ PRODUCT_COPY_FILES += \
     vendor/google/raven/proprietary/vendor/bin/cbd:$(TARGET_COPY_OUT_VENDOR)/bin/cbd \
     vendor/google/raven/proprietary/vendor/bin/chre:$(TARGET_COPY_OUT_VENDOR)/bin/chre \
     vendor/google/raven/proprietary/vendor/bin/dmd:$(TARGET_COPY_OUT_VENDOR)/bin/dmd \
-    vendor/google/raven/proprietary/vendor/bin/hw/android.hardware.biometrics.fingerprint-service.goodix:$(TARGET_COPY_OUT_VENDOR)/bin/hw/android.hardware.biometrics.fingerprint-service.goodix \
     vendor/google/raven/proprietary/vendor/bin/hw/android.hardware.bluetooth@1.1-service.bcmbtlinux:$(TARGET_COPY_OUT_VENDOR)/bin/hw/android.hardware.bluetooth@1.1-service.bcmbtlinux \
     vendor/google/raven/proprietary/vendor/bin/hw/android.hardware.confirmationui@1.0-service.trusty.vendor:$(TARGET_COPY_OUT_VENDOR)/bin/hw/android.hardware.confirmationui@1.0-service.trusty.vendor \
     vendor/google/raven/proprietary/vendor/bin/hw/android.hardware.drm-service.widevine:$(TARGET_COPY_OUT_VENDOR)/bin/hw/android.hardware.drm-service.widevine \
@@ -1455,10 +1414,8 @@ PRODUCT_COPY_FILES += \
     vendor/google/raven/proprietary/vendor/lib64/audio_tunnel_aoc.so:$(TARGET_COPY_OUT_VENDOR)/lib64/audio_tunnel_aoc.so \
     vendor/google/raven/proprietary/vendor/lib64/audio_usb_aoc.so:$(TARGET_COPY_OUT_VENDOR)/lib64/audio_usb_aoc.so \
     vendor/google/raven/proprietary/vendor/lib64/audio_waves_aoc.so:$(TARGET_COPY_OUT_VENDOR)/lib64/audio_waves_aoc.so \
-    vendor/google/raven/proprietary/vendor/lib64/biometricsuez.so:$(TARGET_COPY_OUT_VENDOR)/lib64/biometricsuez.so \
     vendor/google/raven/proprietary/vendor/lib64/com.google.edgetpu_app_service-V2-ndk.so:$(TARGET_COPY_OUT_VENDOR)/lib64/com.google.edgetpu_app_service-V2-ndk.so \
     vendor/google/raven/proprietary/vendor/lib64/com.google.edgetpu_vendor_service-V2-ndk.so:$(TARGET_COPY_OUT_VENDOR)/lib64/com.google.edgetpu_vendor_service-V2-ndk.so \
-    vendor/google/raven/proprietary/vendor/lib64/com.google.hardware.biometrics.fingerprint.fingerprint-ext-V1-ndk.so:$(TARGET_COPY_OUT_VENDOR)/lib64/com.google.hardware.biometrics.fingerprint.fingerprint-ext-V1-ndk.so \
     vendor/google/raven/proprietary/vendor/lib64/com.google.input-V1-ndk.so:$(TARGET_COPY_OUT_VENDOR)/lib64/com.google.input-V1-ndk.so \
     vendor/google/raven/proprietary/vendor/lib64/egl/libGLES_mali.so:$(TARGET_COPY_OUT_VENDOR)/lib64/egl/libGLES_mali.so \
     vendor/google/raven/proprietary/vendor/lib64/fp_utils.so:$(TARGET_COPY_OUT_VENDOR)/lib64/fp_utils.so \
@@ -1540,8 +1497,6 @@ PRODUCT_COPY_FILES += \
     vendor/google/raven/proprietary/vendor/lib64/libsueznanoappclients.so:$(TARGET_COPY_OUT_VENDOR)/lib64/libsueznanoappclients.so \
     vendor/google/raven/proprietary/vendor/lib64/libtouchflow.so:$(TARGET_COPY_OUT_VENDOR)/lib64/libtouchflow.so \
     vendor/google/raven/proprietary/vendor/lib64/libusf.so:$(TARGET_COPY_OUT_VENDOR)/lib64/libusf.so \
-    vendor/google/raven/proprietary/vendor/lib64/libvendor.goodix.hardware.biometrics.fingerprint.so:$(TARGET_COPY_OUT_VENDOR)/lib64/libvendor.goodix.hardware.biometrics.fingerprint.so \
-    vendor/google/raven/proprietary/vendor/lib64/libvendor.goodix.hardware.biometrics.fingerprint@2.1.so:$(TARGET_COPY_OUT_VENDOR)/lib64/libvendor.goodix.hardware.biometrics.fingerprint@2.1.so \
     vendor/google/raven/proprietary/vendor/lib64/libwlcproto_13.so:$(TARGET_COPY_OUT_VENDOR)/lib64/libwlcproto_13.so \
     vendor/google/raven/proprietary/vendor/lib64/libwvaidl.so:$(TARGET_COPY_OUT_VENDOR)/lib64/libwvaidl.so \
     vendor/google/raven/proprietary/vendor/lib64/sensors.usf.so:$(TARGET_COPY_OUT_VENDOR)/lib64/sensors.usf.so \
-- 
2.39.2

