From 6a69e27716d58a32f56aa359a0a6f7a540b18910 Mon Sep 17 00:00:00 2001
From: danielk43 <11856117+danielk43@users.noreply.github.com>
Date: Sun, 16 Oct 2022 00:50:07 -0400
Subject: [PATCH] bramble: Disable mainline checking

---
 aosp_bramble.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/aosp_bramble.mk b/aosp_bramble.mk
--- a/aosp_bramble.mk
+++ b/aosp_bramble.mk
@@ -25,7 +25,7 @@ endif
 $(call inherit-product, $(SRC_TARGET_DIR)/product/generic_system.mk)
 
 # Enable mainline checking
-PRODUCT_ENFORCE_ARTIFACT_PATH_REQUIREMENTS := strict
+#PRODUCT_ENFORCE_ARTIFACT_PATH_REQUIREMENTS := strict
 
 #
 # All components inherited here go to system_ext image

