From 117dc62530fbe864b8ee20c50aa8e87b7e2eba3c Mon Sep 17 00:00:00 2001
From: danielk43 <11856117+danielk43@users.noreply.github.com>
Date: Sun, 16 Oct 2022 02:09:43 -0400
Subject: [PATCH 3/3] Move avb key to device bramble

---
 BoardConfig-common.mk | 2 --
 BoardConfigLineage.mk | 3 ---
 2 files changed, 5 deletions(-)

diff --git a/BoardConfig-common.mk b/BoardConfig-common.mk
--- a/BoardConfig-common.mk
+++ b/BoardConfig-common.mk
@@ -107,8 +107,6 @@ TARGET_RECOVERY_UI_LIB := \
 
 # Enable chain partition for system.
 BOARD_AVB_VBMETA_SYSTEM := system system_ext product
-BOARD_AVB_VBMETA_SYSTEM_KEY_PATH := external/avb/test/data/testkey_rsa2048.pem
-BOARD_AVB_VBMETA_SYSTEM_ALGORITHM := SHA256_RSA2048
 BOARD_AVB_VBMETA_SYSTEM_ROLLBACK_INDEX := $(PLATFORM_SECURITY_PATCH_TIMESTAMP)
 BOARD_AVB_VBMETA_SYSTEM_ROLLBACK_INDEX_LOCATION := 1
 
diff --git a/BoardConfigLineage.mk b/BoardConfigLineage.mk
--- a/BoardConfigLineage.mk
+++ b/BoardConfigLineage.mk
@@ -50,6 +50,3 @@ endif
 # SELinux
 BOARD_SEPOLICY_DIRS += device/google/redbull/sepolicy-lineage/dynamic
 BOARD_SEPOLICY_DIRS += device/google/redbull/sepolicy-lineage/vendor
-
-# Verified Boot
-BOARD_AVB_MAKE_VBMETA_IMAGE_ARGS += --flags 3

