From 54c7d3602e1bcd9f48473c5b328987b64f79f03c Mon Sep 17 00:00:00 2001
From: danielk43 <11856117+danielk43@users.noreply.github.com>
Date: Sun, 29 Oct 2023 21:12:07 -0400
Subject: [PATCH 2/2] panther: Update apps

---
 Android.bp        | 22 ----------------------
 panther-vendor.mk |  2 --
 2 files changed, 24 deletions(-)

diff --git a/Android.bp b/Android.bp
index 8c304d7..284013e 100644
--- a/Android.bp
+++ b/Android.bp
@@ -5,28 +5,6 @@
 soong_namespace {
 }
 
-android_app_import {
-	name: "PixelCameraServicesP10",
-	owner: "google",
-	apk: "proprietary/product/app/PixelCameraServicesP10/PixelCameraServicesP10.apk",
-	presigned: true,
-	dex_preopt: {
-		enabled: false,
-	},
-	product_specific: true,
-}
-
-android_app_import {
-	name: "VZWAPNLib",
-	owner: "google",
-	apk: "proprietary/product/app/VZWAPNLib/VZWAPNLib.apk",
-	presigned: true,
-	dex_preopt: {
-		enabled: false,
-	},
-	product_specific: true,
-}
-
 android_app_import {
 	name: "CarrierSettings",
 	owner: "google",
diff --git a/panther-vendor.mk b/panther-vendor.mk
index 0c89f9a..59076eb 100644
--- a/panther-vendor.mk
+++ b/panther-vendor.mk
@@ -67,8 +67,6 @@ PRODUCT_COPY_FILES += \
     vendor/google/panther/proprietary/system_ext/lib64/vendor.samsung_slsi.telephony.hardware.radioExternal@1.0.so:$(TARGET_COPY_OUT_SYSTEM_EXT)/lib64/vendor.samsung_slsi.telephony.hardware.radioExternal@1.0.so \
 
 PRODUCT_PACKAGES += \
-    PixelCameraServicesP10 \
-    VZWAPNLib \
     CarrierSettings \
     OemRilHookService \
     OemRilService \
-- 
2.39.2

